docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.html

```diff
@@ -0,0 +1,42 @@
+<button uxFocusIndicator
+        tabindex="0"
+        class="icon-demo btn btn-link button-secondary btn-text"
+        [uxPopover]="iconCodeSnippet"
+        popoverClass="icon-code-snippets"
+        placement="bottom">
+    <i class="{{ iconClass }} p-r-xs" [ngClass]="classname"></i>
+    <span>{{ iconset === 'ux-icons' ? name : classname }}</span>
+</button>
+
+<ng-template #iconCodeSnippet>
+    <table class="icon-code-snippets-table">
+        <tbody>
+        <tr *ngIf="iconset === 'ux-icons'">
+            <td class="p-l-xs">&lt;ux-icon name="{{name}}"&gt;&lt;/ux-icon&gt;&nbsp;&nbsp;</td>
+            <td><i class="ux-icon ux-icon-clipboard m-r-xs"
+                   [uxTooltip]="tooltipTemplate"
+                    (click)="copy('ux-icon-component')"></i></td>
+        </tr>
+        <tr *ngIf="iconset === 'ux-icons'">
+            <td class="p-l-xs">&lt;i class="ux-icon {{classname}}"&gt;&lt;/i&gt;&nbsp;&nbsp;</td>
+            <td><i class="ux-icon ux-icon-clipboard m-r-xs"
+                   [uxTooltip]="tooltipTemplate"
+                   (click)="copy('ux-icon')"></i></td>
+        </tr>
+        <tr *ngIf="iconset !== 'ux-icons'">
+            <td class="p-l-xs">&lt;i class="hpe-icon {{classname}}"&gt;&lt;/i&gt;&nbsp;&nbsp;</td>
+            <td><i class="ux-icon ux-icon-clipboard m-r-xs"
+                   [uxTooltip]="tooltipTemplate"
+                    (click)="copy('hpe-icon')"></i></td>
+        </tr>
+        </tbody>
+    </table>
+    <p class="usage-text" *ngIf="iconset === 'ux-icons'">The latter form is recommended only if Angular is
+        not available.
+    </p>
+</ng-template>
+
+<ng-template #tooltipTemplate>
+    <p class="usage-tooltip">Copy to Clipboard</p>
```

Comment:
Looks like this `p` tag has margin on the bottom making the popover have empty space below:

![image](https://user-images.githubusercontent.com/20795331/62369946-1d37e600-b529-11e9-8983-c1f97caaaa4a.png)

Reply:
I'd suggest only using `p` for actual paragraphs of text, otherwise a div is more appropriate.

Reply:
changed to <code>div</code> rather than <code>p</code>